api/ceramic.yaml

```diff
@@ -495,3 +575,17 @@ components:
         streamId:
           type: string
           description: Multibase encoded stream ID.
+    RefreshTokenRequest:
+      title: A request for a refresh token
+      type: object
+      properties:
```

Comment:
should the `requestId` property be required on this and AccessTokenRequest?

Reply:
At this point, I don't have enough information to really put anything in this structure, so would like to leave it empty, but openapi doesn't allow that. So I added requestId as an optional parameter, for tracing, even though that might be better as a header.

I feel like I might need list of resources and actions for a refresh token, but wasn't confident enough to add them in this PR.